.github/workflows/MainDistributionPipeline.yml

```
@@ -20,11 +21,22 @@ jobs:
+      ci_tools_version: v1.3.2
+      extension_name: fastlanes
+      exclude_archs: 'wasm_mvp;wasm_eh;wasm_threads;windows_amd64_rtools;windows_amd64_mingw;linux_amd64_musl;linux_amd64_gcc4'
+
+  code-quality-check:
+    name: Code Quality Check
+    uses: duckdb/extension-ci-tools/.github/workflows/_extension_code_quality.yml@main
+    with:
+      duckdb_version: v1.3.2
+      ci_tools_version: main
+      extension_name: quack
+      format_checks: 'format;tidy'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)